crates/compute_test_utils/src/layer.rs

```diff
@@ -0,0 +1,616 @@
+// Copyright 2025 Irreducible Inc.
```

Comment:
Without checking, I assume that those `fn`s are just copied from the original location without modification?

Reply:
Yes, that's right